safe/gui/tools/wizard/step_fc90_analysis.py

```diff
@@ -259,3 +261,54 @@ def progress_callback(self, current_value, maximum_value, message=None):
         self.progress_bar.setMaximum(maximum_value)
         self.progress_bar.setValue(current_value)
         QtGui.QApplication.processEvents()
+
+    def print_map(self):
+        """Open impact report dialog used to tune report when printing."""
+        # Check if selected layer is valid
+        impact_layer = self.parent.iface.activeLayer()
+        if impact_layer is None:
+            # noinspection PyCallByClass,PyTypeChecker
+            QtGui.QMessageBox.warning(
+                self,
+                self.tr('InaSAFE'),
+                self.tr('Please select a valid impact layer before '
+                        'trying to print.'))
+            return
+
+        # Get output path from datastore
+        # Fetch report for pdfs report
+        report_path = os.path.dirname(impact_layer.source())
+        output_paths = [
+            os.path.join(
+                report_path,
+                'output/impact-report-output.pdf'),
+            os.path.join(
+                report_path,
+                'output/a4-portrait-blue.pdf'),
+            os.path.join(
+                report_path,
+                'output/a4-landscape-blue.pdf'),
+        ]
+
+        # Make sure the file paths can wrap nicely:
+        wrapped_output_paths = [
+            path.replace(os.sep, '<wbr>' + os.sep) for path in
```

Comment:
http://www.w3schools.com/Tags/tag_wbr.asp
The <wbr> (Word Break Opportunity) tag specifies where in a text it would be ok to add a line-break.